library/src/main/java/in/krharsh17/barview/ViewUtils.java

```diff
@@ -0,0 +1,64 @@
+package in.krharsh17.barview;
+
+import android.graphics.Paint;
+import android.graphics.Rect;
+import android.graphics.drawable.Drawable;
+import android.graphics.drawable.LayerDrawable;
+import android.graphics.drawable.ShapeDrawable;
+import android.graphics.drawable.shapes.RoundRectShape;
+import android.view.Gravity;
+import android.view.View;
+
+import androidx.annotation.ColorRes;
+import androidx.annotation.DimenRes;
+import androidx.core.content.ContextCompat;
+
+import static android.view.View.LAYER_TYPE_SOFTWARE;
+
+/**
+ * Created by ArmanSo on 4/16/17.
+ */
```

Comment:
Please remove this

Reply:
View Utils file is not needed so we can delete this